doc/introduction/templates.rst

```diff
@@ -271,85 +297,5 @@ respective interfaces.
     init_torch = torch.tensor(init_pars)
     init_tf = tf.Variable(init_pars)
 
-The following initialization functions are available:
-
-.. rubric:: Continuous-variable quantum neural network
-
-:html:`<div class="summary-table">`
-
-.. autosummary::
-    :nosignatures:
-
-    ~pennylane.init.cvqnn_layers_all
-    ~pennylane.init.cvqnn_layers_theta_uniform
-    ~pennylane.init.cvqnn_layers_theta_normal
-    ~pennylane.init.cvqnn_layers_phi_uniform
-    ~pennylane.init.cvqnn_layers_phi_normal
-    ~pennylane.init.cvqnn_layers_varphi_uniform
-    ~pennylane.init.cvqnn_layers_varphi_normal
-    ~pennylane.init.cvqnn_layers_r_uniform
-    ~pennylane.init.cvqnn_layers_r_normal
-    ~pennylane.init.cvqnn_layers_phi_r_uniform
-    ~pennylane.init.cvqnn_layers_phi_r_normal
-    ~pennylane.init.cvqnn_layers_a_uniform
-    ~pennylane.init.cvqnn_layers_a_normal
-    ~pennylane.init.cvqnn_layers_phi_a_uniform
-    ~pennylane.init.cvqnn_layers_phi_a_normal
-    ~pennylane.init.cvqnn_layers_kappa_uniform
-    ~pennylane.init.cvqnn_layers_kappa_normal
-
-
-:html:`</div>`
-
-.. rubric:: Interferometer
-
-:html:`<div class="summary-table">`
-
-.. autosummary::
-    :nosignatures:
-
-    ~pennylane.init.interferometer_all
-    ~pennylane.init.interferometer_theta_uniform
-    ~pennylane.init.interferometer_theta_normal
-    ~pennylane.init.interferometer_phi_uniform
-    ~pennylane.init.interferometer_phi_normal
-    ~pennylane.init.interferometer_varphi_uniform
-    ~pennylane.init.interferometer_varphi_normal
-
-:html:`</div>`
-
-.. rubric:: QAOA embedding
-
-:html:`<div class="summary-table">`
-
-.. autosummary::
-    :nosignatures:
-
-    ~pennylane.init.qaoa_embedding_uniform
-    ~pennylane.init.qaoa_embedding_normal
-
-:html:`</div>`
-
-.. rubric:: Random layers
-
-:html:`<div class="summary-table">`
-
-.. autosummary::
-    :nosignatures:
-
-    ~pennylane.init.random_layers_uniform
-    ~pennylane.init.random_layers_normal
-
-:html:`</div>`
-
-.. rubric:: Strongly entangling layers
-
-:html:`<div class="summary-table">`
-
-.. autosummary::
-    :nosignatures:
-
-    ~pennylane.init.strong_ent_layers_uniform
-    ~pennylane.init.strong_ent_layers_normal
+The initialization functions can be found in the :mod:`~.pennylane.init` module.
```

Comment:
One thing that came to my mind here: could you perhaps organize this small change into a separate PR? I'm super happy to approve it right away :blush: 

One of the reasons for this would be cutting down the size of this one, such that more emphasis can be put on viewing and reviewing the changes related to `broadcast`. Also, it would be easy to find this small change in the list of merged PRs. :slightly_smiling_face:

Reply:
You are so right :)